coursier.bzl

```diff
@@ -510,7 +510,10 @@ def _coursier_fetch_impl(repository_ctx):
         repository_ctx.file("exclusion-file.txt", "\n".join(exclusion_lines), False)
         cmd.extend(["--local-exclude-file", "exclusion-file.txt"])
     for repository in repositories:
-        cmd.extend(["--repository", utils.repo_url(repository)])
+        cmd.extend(["--repository", repository["repo_url"]])
+    for repository in repositories:
+        if "credentials" in repository:
+            cmd.extend(["--credentials", utils.repo_credentials(repository)])
```

Comment:
Do you need quotes around `util.repo_credentials(repository)`?  https://get-coursier.io/docs/other-credentials.html#inline suggests so.

Reply:
The linked article shows usage through bash, we are passing it to an execution context directly here. A string here should be passed as a single argument to `coursier`, so there is no need to escape it.